website/src/routes/apiv2.js

```diff
@@ -0,0 +1,776 @@
+/*
```

Comment:
Unfortunately, no. The work I did on the content file has not been merged in. I'm going to incrementally swap out the pieces of the content file with the ES6 versions.